src/components/button/Button.css

```diff
@@ -14,7 +15,11 @@
   font-size: 16px;
   cursor: pointer;
 }
-
+.main-button:hover {
+  background-color: #ffffff;
+  color: black;
+  transition: ease-in 0.3s;
+}
```

Comment:
Do you think this hover effect is necessary

Reply:
In my opinion, yes. Some simple lightweight transitions can make a page feel a lot smoother and interactive. Also it fits in with the theme too but it's your repo so it's your call mate.
Remove the commit if you don't want to add it.

Reply:
Bro's it's your Portfolio also it's your choice also and i think you made your point clear and I love it 
Thanks For Contributing